h2o-extensions/xgboost/build.gradle

```diff
@@ -1,7 +1,7 @@
 description = "H2O XGBoost support"
 
 ext {
-    xgboost4jVersion = "1.2.0.16"
+    xgboost4jVersion = "1.6.1.13-kn-merge_1.6.0-SNAPSHOT"
```

Comment:
Shouldn't we use a non-snapshot version?

Reply:
It will, after we release: https://github.com/h2oai/xgboost/pull/93

Reply:
Done